traffic_portal/test/end_to_end/conf.json

```diff
@@ -27,7 +27,8 @@
       "servers/servers-spec.js",
       "topologies/topologies-spec.js",
       "deliveryServices/delivery-services-spec.js",
-      "jobs/jobs-spec.js"
+      "jobs/jobs-spec.js",
+      "tenants/tenants-spec.js"
```

Comment:
This needs to be added to `.github/actions/tp-e2e-tests/conf.json`, although in a PR to fix some other issues I'll probably change it so that it just uses this `conf.json`

Reply:
ok, so want me to leave this to your new PR?

Reply:
I'd add it here so I don't have to run the TP tests locally :laughing:.

Reply:
done